### DISCUSSION TITLE
Theme's Module Overrides Site Defaults

### DISCUSSION THREAD

**Discussion by ghost** [colinwilson/lotusdocs#26] [Q&A] [answered]
From the [quickstart guide](https://lotusdocs.dev/docs/quickstart/#create-a-new-lotus-docs-site):

> This will create a markdown file named `example-page.md` with the following default front matter:

```
+++
title = "Example Page"
description = ""
icon = "article"
date = "2023-05-22T00:27:57+01:00"
lastmod = "2023-05-22T00:27:57+01:00"
draft = false
toc = true
weight = 999
+++
```

It looks like this overrides the default archetype - `my-docs-site/archetypes/default.md`.

Is it common for themes initialized as modules to do this ? Is there a way to prefer default values in directory `my-docs-site/` over the imported module ?

---

**Edit 2:** Seeing the same behavior for `favicon.ico` - I've added the file in `my-docs-site/static/favicon.ico`, but it's not reflected in the browser's title bar.

A few links that I've gone through on Hugo's Discourse indicate that the actual site's settings override the theme's. 

Is there something I need to do that isn't in the [quickstart guide](https://lotusdocs.dev/docs/quickstart/)?

---

**Edit 3:** Archetype `my-docs-site/archetypes/docs.md` overrides the theme's archetype. Unable to override the `favicon` though.

**Comment by colinwilson** [comment, marked as answer]
The path to the `/docs` favicon is `my-docs-site/static/docs/favicon.ico`. But this it temporary. The path will be updated soon, so that both the landing page and docs section use the root favicon files.
